### PR TITLE
Update project and Google Tag Manager

### PR DESCRIPTION
**Pull request by SantosGuillamot**
I've updated the project following[ this guide](https://docs.frontity.org/guides/keep-frontity-updated) and I've installed the new version of `@frontity/google-tag-manager-analytics`.

Note that, as we don't have the latest version of the Gutenberg plugin in our current web (it would break some things), the preview is going to have some weird buttons that will be solved after the release according to this other [Pull Request](https://github.com/frontity/frontity.org/pull/126). I'll update the plugin just after releasing this new version.

### CODE REVIEW

**Comment by frontibotito** [comment]
Deploy preview for _website_ ready!

Built with commit 337bccb863799bb1502f4d83e0d73bcbe828796d

https://frontity-org-9z85kz3tn.vercel.app

File: frontity.settings.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ const settings = [
         name: "@frontity/wp-source",
         state: {
           source: {
-            api: "https://frontity.org/wp-json",
+            api: "https://dev-frontity-org.pantheonsite.io/wp-json",
```
Comment (DAreRodz) [inline frontity.settings.js:19]: I think this is a mistake, right? :sweat_smile:

File: package.json [hunk 2]
```diff
@@ -18,28 +18,29 @@
...
     "@changesets/changelog-github": "^0.2.6",
-    "@changesets/cli": "^2.9.1",
-    "@frontity/core": "^1.7.1",
+    "@changesets/cli": "^2.9.2",
+    "@frontity/core": "^1.7.2",
     "@frontity/frontity-org-theme": "file:packages/frontity-org-theme",
-    "@frontity/google-analytics": "^1.1.0",
-    "@frontity/google-tag-manager": "^0.2.0",
+    "@frontity/google-analytics": "^1.2.0",
+    "@frontity/google-tag-manager": "^0.3.0",
```
Comment (DAreRodz) [inline package.json:25]: We are not using the `@frontity/google-tag-manager` package, are we? I guess you have to remove it from `package.json`.

File: frontity.settings.js (hunk 1 above)
Comment (SantosGuillamot) [reply frontity.settings.js:19]: You're right :smile: I was testing the new version of the Gutenberg plugin.

File: package.json (hunk 2 above)
Comment (SantosGuillamot) [reply package.json:25]: Yes, I removed it from the `frontity.settings.js` and forgot to delete it here.

**Pull request merged**